### PR TITLE
prepare for release v1.0.5

### PR DESCRIPTION
**Pull request by ianvernon**
Signed-off-by: Ian Vernon <ian@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4884)

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-missed-k8s

**Comment by ianvernon** [comment]
K8s tests failed:
```
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
WorkflowScript: 137: expecting '}', found '' @ line 137, column 1.
1 error

	at org.codehaus.groovy.control.ErrorCollector.failIfErrors(ErrorCollector.java:310)
	at org.codehaus.groovy.control.ErrorCollector.addFatalError(ErrorCollector.java:150)
	at org.codehaus.groovy.control.ErrorCollector.addError(ErrorCollector.java:120)
	at org.codehaus.groovy.control.ErrorCollector.addError(ErrorCollector.java:132)
	at org.codehaus.groovy.control.SourceUnit.addError(SourceUnit.java:350)
	at org.codehaus.groovy.antlr.AntlrParserPlugin.transformCSTIntoAST(AntlrParserPlugin.java:144)
	at org.codehaus.groovy.antlr.AntlrParserPlugin.parseCST(AntlrParserPlugin.java:110)
	at org.codehaus.groovy.control.SourceUnit.parse(SourceUnit.java:234)
	at org.codehaus.groovy.control.CompilationUnit$1.call(CompilationUnit.java:168)
	at org.codehaus.groovy.control.CompilationUnit.applyToSourceUnits(CompilationUnit.java:943)
	at org.codehaus.groovy.control.CompilationUnit.doPhaseOperation(CompilationUnit.java:605)
	at org.codehaus.groovy.control.CompilationUnit.processPhaseOperations(CompilationUnit.java:581)
	at org.codehaus.groovy.control.CompilationUnit.compile(CompilationUnit.java:558)
	at groovy.lang.GroovyClassLoader.doParseClass(GroovyClassLoader.java:298)
	at groovy.lang.GroovyClassLoader.parseClass(GroovyClassLoader.java:268)
	at groovy.lang.GroovyShell.parseClass(GroovyShell.java:688)
	at groovy.lang.GroovyShell.parse(GroovyShell.java:700)
	at org.jenkinsci.plugins.workflow.cps.CpsGroovyShell.doParse(CpsGroovyShell.java:131)
	at org.jenkinsci.plugins.workflow.cps.CpsGroovyShell.reparse(CpsGroovyShell.java:125)
	at org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.parseScript(CpsFlowExecution.java:560)
	at org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.start(CpsFlowExecution.java:521)
	at org.jenkinsci.plugins.workflow.job.WorkflowRun.run(WorkflowRun.java:329)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
Finished: FAILURE
```

cc: @eloycoto @aanm

**Comment by ianvernon** [comment]
K8s Jenkinsfile is broken because of the following:

```
...
           environment {
                FAILFAST=setIfPR("true", "false")
           ## there should be a '}' here
            options {
                timeout(time: 120, unit: 'MINUTES')
            }
...
```

**Comment by aanm** [comment]
@ianvernon probably this commit needs to be backported? https://github.com/cilium/cilium/commit/78499dec6a235d19031d891ab461c9175a38c387 cc @eloycoto

**Comment by eloycoto** [comment]
I don't think so, the master ginkgo Jenkinsfile is ok: 

```
breo.acalustra.com ➜  cilium git:(v1.0) ✗ git rev-parse HEAD
4f02eec5112d721c1afc60367c8e8125d6d11cd1
breo.acalustra.com ➜  cilium git:(v1.0) ✗ curl -X POST -F "jenkinsfile=<ginkgo.Jenkinsfile" http://jenkins.cilium.io:8080/pipeline-model-converter/validate

Jenkinsfile successfully validated.
breo.acalustra.com ➜  cilium git:(v1.0) ✗
```

Any other change?

**Comment by eloycoto** [comment]
test-me-please

**Comment by ianvernon** [comment]
@aanm @eloycoto should this block the release of v1.0.5 ? The normal CI tests work just fine, so I think we can go ahead and release since this bug is only in the Jenkinsfile, not in Cilium itself.

**Comment by aanm** [comment]
@ianvernon yes we should block as this is a release. The fix for your issue is here. https://github.com/cilium/cilium/pull/4907 as soon the tests start the PR can be merged since the issue is only related with the jenkinsfile itself

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-missed-k8s

File: examples/kubernetes/1.9/cilium.yaml [hunk 1]
```diff
@@ -223,7 +223,6 @@ spec:
           secret:
             secretName: cilium-etcd-secrets
       restartPolicy: Always
-      priorityClassName: system-node-critical
```
Comment (aanm) [inline examples/kubernetes/1.9/cilium.yaml]: wow, what's is happening here?

**Review by aanm — CHANGES_REQUESTED** [review]

File: examples/kubernetes/1.9/cilium.yaml (hunk 1 above)
Comment (aanm) [reply examples/kubernetes/1.9/cilium.yaml]: You probably did `make` in `examples/kubernetes`. Can you fix it on a commit and then re-create the files again?

Comment (ianvernon) [reply examples/kubernetes/1.9/cilium.yaml]: This is part of the release notes (step 6) - http://docs.cilium.io/en/v1.1/contributing/#generic-release-process

Comment (ianvernon) [reply examples/kubernetes/1.9/cilium.yaml]: This is odd though, how did this happen? Is there something I can backport here?

Comment (ianvernon) [reply examples/kubernetes/1.9/cilium.yaml]: Manually fixed in latest upload.

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-missed-k8s

**Review by aanm — APPROVED** [review]

**Pull request merged**